deploy/helm-chart/servicegateway/templates/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "servicegateway.fullname" . }}
+  labels:
+        {{ include "servicegateway.labels" . | indent 4 }}
```

Comment:
The indentation looks... odd.

Reply:
Also, do we need the [helm standard labels](https://helm.sh/docs/chart_best_practices/#standard-labels) on this?

Reply:
Even though they're not mandatory, it's still [recommended by Helm](https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md#standard-labels) to use them.

Reply:
If they're recommended, we should set them.